src/main/java/no/sikt/nva/pubchannels/utils/ApplicationConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package no.sikt.nva.pubchannels.utils;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.databind.JsonNode;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import software.amazon.awssdk.services.appconfig.AppConfigClient;
+import software.amazon.awssdk.services.appconfig.model.GetConfigurationRequest;
+
+public class ApplicationConfiguration {
```

Comment:
Can you make this implement an interface called ApplicationConfiguration. 

Replace the mocked version with a fake where relevant.